code/modules/clothing/outfits/ert.dm

```diff
@@ -537,3 +537,111 @@
 	name = "Death Commando Officer"
 
 	head = /obj/item/clothing/head/helmet/space/beret
+
+/datum/outfit/centcom/ert/marine
+	name = "Marine Commander"
+
+	id = /obj/item/card/id/advanced/centcom/ert
+	suit = /obj/item/clothing/suit/armor/vest/marine
+	suit_store = /obj/item/gun/ballistic/automatic/wt550
+	back = /obj/item/shield/riot
+	belt = /obj/item/storage/belt/military/assault/full
+	glasses = /obj/item/clothing/glasses/hud/security/sunglasses/eyepatch
+	l_pocket = /obj/item/kitchen/knife/combat
+	r_pocket = /obj/item/lighter
+	uniform = /obj/item/clothing/under/syndicate/camo
+	mask = /obj/item/clothing/mask/cigarette/robustgold
+	head = /obj/item/clothing/head/helmet/marine
+	l_hand = /obj/item/flashlight/seclite
+
+/datum/outfit/centcom/ert/marine/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	..()
+
+	if(visualsOnly)
+		return
+	var/obj/item/radio/R = H.ears
+	R.keyslot = new /obj/item/encryptionkey/heads/captain
+	R.recalculateChannels()
+
+/datum/outfit/centcom/ert/marine/security
+	name = "Marine Heavy"
+
+	id = /obj/item/card/id/advanced/centcom/ert/security
+	suit = /obj/item/clothing/suit/armor/vest/marine/security
+	suit_store = /obj/item/gun/ballistic/automatic/wt550
+	back = /obj/item/shield/riot
+	belt = /obj/item/storage/belt/military/assault/full
+	glasses = /obj/item/clothing/glasses/hud/security/sunglasses
+	r_pocket = /obj/item/grenade/smokebomb
+	mask = /obj/item/clothing/mask/gas/sechailer
+	head = /obj/item/clothing/head/helmet/marine/security
+	l_hand = /obj/item/flashlight/seclite
+
+/datum/outfit/centcom/ert/marine/security/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
```

Comment:
please use better variable names

https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names